peachjam/adapters/adapters.py

```diff
@@ -104,7 +110,28 @@ def update_document(self, url):
             "work": work,
         }
 
-        doc, _ = Legislation.objects.update_or_create(
+        if document["nature"] == "act":
+            if (
+                document["subtype"] == "charter"
+                or "protocol"
+                or "convention"
+                or "treaty"
+            ):
+                model = LegalInstrument
+                field_data["nature"] = DocumentNature.objects.update_or_create(
+                    name=document["nature"]
```

Comment:
```suggestion
                    name=document["subtype"]
```

The subtype is the more interesting aspect.